packages/css/src/components/viewer-grid/viewer-grid.stories.ts

```diff
@@ -18,18 +77,43 @@ const main = html`
       himenaeos. Quisque sodales nibh risus, ac dictum diam volutpat quis. Integer sed tortor quis libero iaculis
       interdum.
     </p>
-    <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean laoreet ligula vel cursus consequat. Aenean id dolor
-      felis. Mauris eget ullamcorper neque. Donec finibus libero lorem, faucibus sollicitudin dui vehicula eu. Vestibulum
-      libero lorem, rutrum ac blandit euismod, elementum vel diam. Nunc at convallis mi. Aliquam tincidunt ante eu quam
-      molestie, nec rutrum quam accumsan. Pellentesque porta quis sem et dictum. Curabitur varius vel metus non pulvinar.
-      Donec ut magna ut nunc lacinia vestibulum et eget enim. In efficitur felis non massa vulputate, eu pellentesque nisi
-      sagittis. Nulla non magna eros. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos
-      himenaeos. Quisque sodales nibh risus, ac dictum diam volutpat quis. Integer sed tortor quis libero iaculis
-      interdum.
-    </p>
-    <p>Morbi vestibulum eu urna sed lacinia.</p>
   </div>
+  <ul class="dso-document-list">
+    <li>
+      <div class="dso-document-list-item">
```

Comment:
Waarom die tussen-`<div>`? Kan dso-document-list-item niet gelijk op de `li`, of... is die class ueberhaupt wel nodig, gezien `.dso-document-list` op de `ul` ?

Reply:
Het component is "Document List Item". De `ul` en `li` gebruik ik om a la `.row` en `.col-` naar buiten en naar binnen te springen. Ik denk dat we er verstandig aan doen om een component niet te fuseren met het lijst component: Dan kan ie ook enkelvoudig worden ingezet